cfgov/jinja2/v1/_includes/organisms/secondary-navigation.html

```diff
@@ -21,7 +21,7 @@
     <ul class="o-secondary-navigation_list
                o-secondary-navigation_list__parents">
         {%- for item in items %}
-            <li>
+            <li data-nav-is-active="{{ is_nav_active(request.url, item) }}">
```

Comment:
You have to determine if any of the  child nav items are active as well.  I've chosen to use a context function but you could use : 

`{{ request.url.find( item.slug ) > - 1 }}`

It's just riskier because you are dependent upon the uniqueness of the slug.